test/e2e/ctx/ctx.go

```diff
@@ -40,6 +40,14 @@ func (ctx TestContext) Logf(f string, v ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, f, v...)
 }
 
+func (ctx TestContext) Fatalf(f string, v ...interface{}) {
```

Comment:
The failure message can be generated with a format string using `fmt.Sprintf` before passing it to Fail, if necessary. Is the shorthand for generating failure messages worth making this API bigger?

Maybe you could add `Failf(string, ...interface{})` to https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/dsl/dsl.go -- or `Fail(string, ...interface{})` if we wanted to stop using dot-imports for the ginkgo DSL -- but I haven't convinced myself that it's a good suggestion.

Reply:
The test was modified to exclude usage of `Fatalf` for logging altogether. I haven't added this for now but will refer to it if I come across a scenario where gomega assertions won't be of much help.